Assets/MixedRealityToolkit/AudioManager/AudioBank.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections.Generic;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Audio
+{
+    /// <summary>
+    /// A collection of AudioEvents and AudioParameters
+    /// </summary>
+    [CreateAssetMenu]
```

Comment:
Let's add this under the pre-existing MixedRealityToolkit menu under asset/create